docs/versioned_docs/version-1/migration.mdx

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
It occurs to me: we probably don't need a document here at all. Instead, we can just update the sidebar config to link directly to the V2 document (I think). Not critical, but might be a bit nicer.